README.md

```diff
@@ -252,26 +247,25 @@ BTUNE_TRADEOFF="(0.3, 0.1, 0.6)" BTUNE_PERF_MODE=DECOMP BTUNE_TRACE=1 python los
 INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
-TRACE: time load model: 0.000283
+TRACE: time load model: 0.000686
 |    Codec   | Filter | Split | C.Level | C.Threads | D.Threads |  S.Score  |  C.Ratio   |   Btune State   | Readapt | Winner
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0134 |         4x |    CODEC_FILTER |    HARD | W
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0352 |         4x |    CODEC_FILTER |    HARD | W
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0374 |         4x |    CODEC_FILTER |    HARD | W
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0366 |         4x |    CODEC_FILTER |    HARD | -
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0393 |         4x |    CODEC_FILTER |    HARD | W
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0365 |         4x |    CODEC_FILTER |    HARD | -
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0357 |         4x |    CODEC_FILTER |    HARD | -
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0368 |         4x |    CODEC_FILTER |    HARD | -
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0365 |         4x |    CODEC_FILTER |    HARD | -
-|       grok |      0 |     0 |       5 |         4 |         4 |    0.0372 |         4x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |     0.113 |      2.18x |    CODEC_FILTER |    HARD | W
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0767 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0764 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |      0.08 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0816 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0821 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0817 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0844 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0838 |      2.18x |    CODEC_FILTER |    HARD | -
+|       zstd |     36 |     1 |       3 |        16 |        16 |    0.0849 |      2.18x |    CODEC_FILTER |    HARD | -
 ```
 In this case, because the quality is more important than before, Btune predicts
-the `blosc2_grok` codec with compression ratio 4x, which losses less info than
-the 8x.
+the `zstd` codec with the integer truncation filter (id 36), which losses less info than
```

Comment:
Would not be better like this?: "losses less info" -> "achieves better quality"

Reply:
Fixed in [dab3e43](https://github.com/Blosc/Blosc2-Btune/pull/13/commits/dab3e4349e75689f32da7aef179a54f38241ab3d).